extension/src/config/config.js

```diff
@@ -0,0 +1,55 @@
+const _ = require('lodash')
+
+const configPath = process.env.CONFIG_PATH
+
+function getEnvConfig () {
+  return {
+    port: process.env.PORT,
+    logLevel: process.env.LOG_LEVEL,
+    apiExtensionBaseUrl: process.env.API_EXTENSION_BASE_URL
+  }
+}
+
+function getCTPEnvCredentials () {
+  return {
+    projectKey: process.env.CTP_PROJECT_KEY,
+    clientId: process.env.CTP_CLIENT_ID,
+    clientSecret: process.env.CTP_CLIENT_SECRET,
+    apiUrl: process.env.CTP_API_URL,
+    authUrl: process.env.CTP_AUTH_URL
```

Comment:
I think we can hardcode API and Auth URLs. They will never change

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/5/commits/bf2f7227796b91c52b6e104ebb7c5fa8d31ff68d

Reply:
No sure we need it global either. I think you can hardcode it on the client creation step

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/5/commits/a5f5edc41e3a31d7fcdb78b20515a5b131b9446a#diff-bb1f7f72b7640d409011bd3d2371b1b7